driver/modern_bpf/programs/attached/dispatchers/syscall_exit.bpf.c

```diff
@@ -7,6 +7,10 @@
 
 #include <helpers/interfaces/syscalls_dispatcher.h>
 #include <helpers/interfaces/attached_programs.h>
+#include <bpf/bpf_helpers.h>
+
+#define X86_64_NR_EXECVE        59
```

Comment:
Can we use the `__NR_*` macros here?

Reply:
assuming building for 64bit, then I think you can use them.